packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/tests/ReviewWorkflows.test.js

```diff
@@ -124,4 +138,83 @@ describe('Admin | Settings | Review Workflow | ReviewWorkflowsPage', () => {
     expect(getByText('2 stages')).toBeInTheDocument();
     expect(saveButton.hasAttribute('disabled')).toBeFalsy();
   });
+
+  test('Successful Stage update', async () => {
+    const toggleNotification = useNotification();
+    const { getByRole } = setup();
+
+    await user.click(
+      getByRole('button', {
+        name: /add new stage/i,
+      })
+    );
+
+    fireEvent.change(getByRole('textbox', { name: /stage name/i }), {
```

Comment:
IMO I think because you have `userEvent` you should continue to do that instead of mixing the two...